### DISCUSSION TITLE
adding new channels

### DISCUSSION THREAD

**Discussion by IKM00** [iptv-org/community#542] [Q&A]
who can help me I'm having trouble adding TV channels, I'm stuck downloading and reading the new channel sheet added

**Comment by ThetaX55** [comment]
what program, app you using?

**Reply by Toro4949** [reply]
Sabes listas para wiseplay?

**Comment by PopeyeTheSai10r** [comment]
See https://github.com/iptv-org/iptv#how-to-use
